tests/runner.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 require( '@babel/register' )( {
-	ignore: /\/node_modules\/(?!@automattic\/dops-components\/)/
+	ignore: [ /\/node_modules\/(?!@automattic\/dops-components\/)/ ],
```

Comment:
I think the reference to `dops-components` here is stale (we don't install that dependency anymore AFAICT). All we need here is just `require( '@babel/register' );`